libs/sdk-ui-kit/styles/scss/header.scss

```diff
@@ -817,6 +817,8 @@ button.gd-button-primary {
     border-radius: 3px;
     color: #fff;
     background: #8351a4;
+    border-color: #000000;
```

Comment:
I think it should be `border: 1px solid rgba(0, 0, 0, 0.1);`

Reply:
At line: https://github.com/gooddata/gooddata-ui-sdk/blob/a0dc1c84d6c866ddc1b67b15716b257ac5e6cb16/libs/sdk-ui-kit/styles/scss/header.scss#L807
Please help me change to:
`button.gd-button-primary.gd-upsell-button`

This will increase the priority of style to avoid override CSS of other classes. (eg: Modeler)
@thuytran-gooddata  @anvuong119 @vannguyengd

Reply:
The upsell button won't display in mobile view port, I think we should add the condition to hide the button when the viewport is smaller than 1024px.

Reply:
As I discussed with Lukas before, he doesn't want the upsell button display on mobile viewport, so we should hide it.